src/app/offers/verify-sign-offer/verify-sign-offer.component.ts

```diff
@@ -71,10 +71,15 @@ export class VerifySignOfferComponent implements OnInit {
             title: 'Transaction signed',
             text: 'Transaction is being processed...',
             footer: 'Check your transaction status<a href="/dash/wallet">&nbsp;here</a>'
+        }).then(() => {
+            this.navigateToWalletScreen();
```

Comment:
Make this oneliner. You don't need to extract a route navigation to a new method. You can navigate in `then` callback directly.